### DISCUSSION TITLE
Dangerzone.rocks link for Debian/Ubuntu takes me to a branch with 0.7.1 versions

### DISCUSSION THREAD

**Discussion by DeltaEpsilon19498** [freedomofpress/dangerzone#1002] [General]
if I go to dangerzone.rocks and click on the button for debian/ubuntu instructions, I get https://github.com/freedomofpress/dangerzone/blob/v0.8.0/INSTALL.md#ubuntu-debian . If I click on "see instructions for mac" or "see instructions for windows," I get https://github.com/freedomofpress/dangerzone/blob/v0.8.0/README.md#macos and https://github.com/freedomofpress/dangerzone/blob/v0.8.0/README.md#windows respectively. Despite the url link name. these links point to the 0.7.1 versions. They do not point to the README in the main branch, which has the 0.8.0 version.

**Comment by apyrgio** [comment]
Dang, you're right. We need to change our links to point to the relevant GitHub release, not our README. I'll send a PR, but due to the nature of Git versions, we'll see a difference from the next version.

**Comment by almet** [comment]
I've created issue #1003 to track this. Thanks for opening this discussion @DeltaEpsilon19498!
